sdk/eventhub/azure-eventhub/tests/livetest/asynctests/test_send_async.py

```diff
@@ -543,7 +543,7 @@ async def on_success(events, pid):
 @pytest.mark.parametrize("keep_alive", [None, 30, 60])
 @pytest.mark.liveTest
 @pytest.mark.asyncio
-def test_send_with_keep_alive_async(connstr_receivers, keep_alive, uamqp_transport):
+async def test_send_with_keep_alive_async(connstr_receivers, keep_alive, uamqp_transport):
```

Comment:
Does this not run if it doesn’t have async here?

Reply:
it runs, but there is a warning in the logs about it that its marked as `pytest.asyncio` but isnt

Reply:
in that case, would it be better to remove that asyncio pytest marker?